src/tests/compiler/ast/module/sourceFileTests.ts

```diff
@@ -1405,4 +1419,247 @@ const t = 5;`;
             });
         });
     });
+
+    describe(nameof<SourceFile>(l => l.removeUnusedDeclarations), () => {
+
```

Comment:
See how the spacing is in the other functions. I would use a code formatter on the project, but they sometimes annoy me... I want to implement my own code formatter in the future.